### ISSUE TITLE
Project Label (e.g. verified) on Donation Page is not up-to-date

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#304]
That little circle should say "verified" "traceable" or "new" (not "active")... I think this wasn't updated on the frontend when we fixed the little circles before

![image.png](https://images.zenhubusercontent.com/5fdfac16a58523a866ddb68d/f536011d-10f9-459c-bd12-bab4958d7c78)

**Comment by RamRamez** [comment]
@laurenluz In /projects we have "New", "Trace" and "VERIFIED"

Should I also change that page?

![image.png](https://images.zenhubusercontent.com/6040fd6c6873c665b22c45b3/9dfbeff8-e529-4655-b25e-2e6e2e2ef475)

**Comment by laurenluz** [comment]
Yes! If you can change the card on the donate page to have the correct label (new, verified, traceable)... the same as it has on the project page... that would be perfect!

**Comment by RamRamez** [comment]
@laurenluz So we don't have "TRACE" label anymore. What about "NEW" label? currently we only show this label for projects made earlier than two weeks ago. Is this good? What about older projects? They shouldn't have any label?

**Comment by laurenluz** [comment]
Ahh yes! It should be:

TRACEABLE (not trace)
NEW (for projects less than 2wks old)
VERIFIED 

Thanks Ramin :)

**Comment by laurenluz** [comment]
For older projects that are not verified, I think no label is fine

**Comment by RamRamez** [comment]
fixed in PR #316

**Comment by DonnaOftadeh** [comment]
@MoeNick @RamRamez should I test it on https://typescript.giveth.io/projects or next?

**Comment by DonnaOftadeh** [comment]
@MoeNick @RamRamez can you please let me know where to test this

**Comment by MoeNick** [comment]
@DonnaOftadeh pls go forward both or them.

**Comment by divine-comedian** [comment]
checking the labels on `next.giveth.io` 

they look to be all updated as per described in this issue - moving to DONE! 
![image](https://user-images.githubusercontent.com/67759413/146589157-8b41c261-ad79-424e-ba46-f54004e778c2.png)
